WoWsShipBuilder.Web/Components/ShipStatsContainer.razor

```diff
@@ -30,6 +30,25 @@
 
 @inherits ReactiveComponentBase<ShipViewModel>
 
+@if (!(initialized && SettingsInitialized))
```

Comment:
Remove the old implementation inside the if(ViewModel is not null) block as it is no longer needed